app/mailers/notification_mailer.rb

```diff
@@ -18,4 +18,22 @@ def return_reminder(item)
     @item = item
     mail(to: '@item.lending.user.email', name: '@item.lending.user.name', subject: "{item.name} is overdue!") # rubocop:disable Rails/I18nLocaleTexts
   end
+
+  def send_notification(message, user, notification_type, as_mail: true)
+    create_notification(message, user, notification_type)
+    mail(to: user.email, subject: "Bookeeper Red Notification") if as_mail # rubocop:disable Rails/I18nLocaleTexts
```

Comment:
Here `Bookeeper Red Notification` has a Typo. Also: Why can't we use the I18n Stuff here?

Reply:
👍🏻

Reply:
It would be great to use translation here. The problem I see, however, is that in difference to displaying a page using a certain language, we here don't know which language the user prefers. I intend to look into it tomorrow.

Reply:
This is a cross-cutting concern. One solution would be persisting the language-preference in the database each time the user toggles it. 

@EliasNotFound (PM) Can you request the mentioned feature to the team which was responsible for the language picker?

Reply:
Fixed the Typo. Internationalization is still an issue.

Reply:
@bweengener I asked team 🌪️ and will keep you updated